neonvm/apis/neonvm/v1/virtualmachine_types.go

```diff
@@ -455,6 +466,37 @@ type VirtualMachine struct {
 	Status VirtualMachineStatus `json:"status,omitempty"`
 }
 
+func (vm *VirtualMachine) GetNetworkUsage(ctx context.Context, timeout time.Duration) (*VirtualMachineNetworkUsage, error) {
```

Comment:
Still thinking we need some way to avoid trying to fetch network usage for old VMs that don't support it. Happy to implement that if it'd be helpful.

Reply:
Won't the requests just fail and it'll be fine? But yeah if there's an easy way to do it that would be good

Reply:
Added a flag to the config, and I tested that its absence disables the network monitoring stuff